### PR TITLE
Dread: Finalize Ghavoran logic revision

### PR DESCRIPTION
**Pull request by XenoWars**
- A few more changes to Spin Boost Tower
- Total Recharge Station North removed the crumble block nodes as they are redundant
- Made it impossible to reach the Screw Attack nodes in the above room and Teleport to Burenia if coming from above since you cannot use Screw Attack while in morph ball
- Added a connection to the Screw Attack node in Total Recharge Station North as there was not one previously
- Added various tricks to connections in Energy Recharge Station
- Added Flash Shift and Spin Boost methods for crossing the crumble blocks in Cross Bomb Tutorial and Teleport to Burenia
- Changed Ballspark to go from Orange to Golzuna Tower proper to beginner movement
- Changed Movement to Stand on Frozen Enemy now that this trick exists for climbing Golzuna Tower with Ice Missiles
- Removed the connection from Start Point to Pulse Radar pickup as this can cause issues if starting here in the future
- Added connection to the powerbeam block below orange as there was not one previously

### CODE REVIEW

File: randovania/games/dread/json_data/Ghavoran.txt [hunk 1]
```diff
@@ -1941,7 +1947,7 @@ Extra - asset_id: collision_camera_024
   * Layers: default
   * Slide Tunnel to Above Pulse Radar/Tunnel to Golzuna Tower
   > Tile Group (POWERBEAM) 1
-      Morph Ball
+      Can Slide
```
Comment (kirbymastah) [inline randovania/games/dread/json_data/Ghavoran.txt:1959]: This distance is too far to reach with sliding alone if i remember correctly - I would keep this as morph ball

File: randovania/games/dread/json_data/Ghavoran.txt [hunk 2]
```diff
@@ -2116,11 +2130,9 @@ Extra - asset_id: collision_camera_025_B
   * Extra - right_shield_entity: Root:pScenario:rEntitiesLayer:dctSublayers:default:dctActors:doorshieldmissile_001
   * Extra - right_shield_def: actordef:actors/props/doorshieldmissile/charclasses/doorshieldmissile.bmsad
   > Door from Chozo Warrior Arena
-      Morph Ball and Before Ghavoran - Ferenia Transport Grapple Box
+      Before Ghavoran - Ferenia Transport Grapple Box and Can Slide
```
Comment (kirbymastah) [inline randovania/games/dread/json_data/Ghavoran.txt:2142]: Same here, this distance is too far to slide alone - keep this as morph ball

File: randovania/games/dread/json_data/Ghavoran.txt (hunk 1 above)
Comment (XenoWars) [reply randovania/games/dread/json_data/Ghavoran.txt:1959]: Without morph ball you just keep sliding until you fall or hit a wall

File: randovania/games/dread/json_data/Ghavoran.txt (hunk 2 above)
Comment (XenoWars) [reply randovania/games/dread/json_data/Ghavoran.txt:2142]: And as before, you'll just keep sliding.

**Review by kirbymastah — CHANGES_REQUESTED** [review]
two small comments about sliding

**Review by kirbymastah — APPROVED** [review]

**Pull request merged**